src/aside/AddRestaurantModal.jsx

```diff
@@ -0,0 +1,47 @@
+import styles from "../css/RestaurantDetailModal.module.css";
+import { options } from "../datas/RestaurantData";
+function AddRestaurantModal() {
```

Comment:
```suggestion
import styles from "../css/RestaurantDetailModal.module.css";
import { options } from "../datas/RestaurantData";

function AddRestaurantModal() {
```

코드의 단락이나 성격이 바뀔 때마다 중간에 줄바꿈을 하듯이, `import` 문과 함수 컴포넌트 사이에도 한 줄을 띄워주셨으면 좋겠습니다~

사실, 공백이나 줄바꿈도 클린 코드에 기여하는 중요한 요소입니다.

Reply:
네넵 명심하겠습니다!!